susemanager-frontend/susemanager-nodejs-sdk-devel/package.json

```diff
@@ -37,7 +36,6 @@
     "eslint-config-react-app": "^5.2.1",
     "eslint-import-resolver-alias": "^1.1.2",
     "eslint-loader": "^3.0.3",
-    "eslint-plugin-flowtype-errors": "^4.4.0",
     "eslint-plugin-flowtype": "^5.2.0",
```

Comment:
Do we still need these other deps then?

Reply:
The preset we use, `extends: ["react-app"]`, relies on `eslint-plugin-flowtype` being available, we just manually don't use it. Hopefully I can clean this out when I get around to https://github.com/SUSE/spacewalk/issues/13515.  

For now, removing it will make ESLint throw, but eventually the plan is to remove this too, yes.